### DISCUSSION TITLE
Component does not detect outside click when the click is on the ReactFlow component

### DISCUSSION THREAD

**Discussion by brilovichgal** [xyflow/xyflow#3912] [Q&A]
Hello,
I have specific component that detects mouse clicks outside of this component.
It works great with all the components except the ReactFlow component, that somehow prevent from this component to detect mouse left-click on it.
The right-click on ReactFlow component is detected, but left-click is not detected.

How to handle that?

**Comment by moklick** [comment]
This is an issue with d3-zoom (which is used for handling zoom and pan). The easiest workaround is to disable pointer-events for the React Flow component when the element that needs the click outside is active.
